examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    u, v = np.mgrid[0:2 * np.pi:30j, 0:np.pi:30j]
</pre><p>Please replace with a call to</p><p><code>plotter = paseos.plot(sim, paseos.PlotType.SpacePlot)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>